Source/UI/NewUI/BillingForm/ViewController/BillingFormViewController.swift

```diff
@@ -135,6 +142,19 @@ final class BillingFormViewController: UIViewController {
                                       keyboardWillShow: #selector(keyboardWillShow),
                                       keyboardWillHide: #selector(keyboardWillHide))
     }
+
+  override func viewWillLayoutSubviews() {
+    super.viewWillLayoutSubviews()
+      if UIKit.UIDevice.current.userInterfaceIdiom == .phone {
+          UIKit.UIDevice.current.setValue(currentOrientation.rawValue, forKey: "orientation")
```

Comment:
I am not a fan of this approach and I think it just adds more complexity and edge cases than it solves. I have tried [wrapping VC in a NavController](https://stackoverflow.com/a/55038584) and using that NavController to wrap this behaviour, which seemed to work well. This is the sample code I quickly threw in Xcode to test

```swift
class PortraitNavigationController: UINavigationController {

    override var modalPresentationStyle: UIModalPresentationStyle {
        get { .fullScreen }
        set { }
    }

    override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
        .portrait
    }

    override var shouldAutorotate: Bool {
        false
    }
}
```

Another serious thing this showed is our reliance on the sample app for the presentation behaviour. Our demo app has a navigation controller that presents the screen. Some consumers may have different architectures and I actually think my suggestion will better address that as well.

Reply:
@alex-ioja-yang-cko thanks for suggestion.
I tried all of these solutions when i started working on it.
1- `NavigationController` inside SDK is not going to work because it cannot be pushed from sample app. 
2- Lock sample project to be portrait mode will be easier solution.
It depends on the release plan. 
@patrick-hoban-cko please advice. Should Sample project be full portrait or only frames?

Reply:
@ehab-al-cko @alex-ioja-yang-cko well if the sample app is using the SDK to create the payment page, then shouldn't the SDK limit the sample app's page to portrait only?

Reply:
>  NavigationController inside SDK is not going to work because it cannot be pushed from sample app.

Are you sure? It worked for me when I tried

> Lock sample project to be portrait mode will be easier solution.

Hard no